src/yunohost/hook.py

```diff
@@ -388,11 +392,27 @@ def hook_exec(path, args=None, raise_on_error=False, no_trace=False,
                 errno.EIO, m18n.n('hook_exec_not_terminated', path=path))
         else:
             logger.error(m18n.n('hook_exec_not_terminated', path=path))
-            return 1
+            return 1, ''
     elif raise_on_error and returncode != 0:
         raise MoulinetteError(
             errno.EIO, m18n.n('hook_exec_failed', path=path))
-    return returncode
+
+    try:
+        with open(stdreturn, 'r') as f:
+            if f.read() != '':
+                returnjson = read_json(stdreturn)
+            else:
+                returnjson = {}
+    except Exception as e:
+        returnjson = {}
```

Comment:
This line seems useless since there is a raise just after that :o

Reply:
Is it ok if I replace this line by this : 
```
os.remove(stdreturn)
os.rmdir(stdreturndir)
```
To fix the next comment too.

Reply:
Should be fixed